.git-blame-ignore-revs

```diff
@@ -17,3 +17,4 @@
 #   from oldest to newest).
 # - Because you must use a hash, you need to append to this list in a follow-up
 #   PR to the actual reformatting PR that you are trying to ignore.
+eebe2461e7e8c697f7fb761b769422716eeed248
```

Comment:
I don't think you should add this right in the PR you're about to merge. Any rebase will make this value obsolete. I'd be strict here and made this a rule: adding a line to `.git-blame-ignore-revs` requires a separate PR. Mistakes will be made without it.

Reply:
Well, the only tricky part here is changing the hash on rebase and merge using a merge commit.
Ok, let's make a separate PR.

> I'd be strict here and made this a rule: adding a commit to .git-blame-ignore-revs requires a separate PR.
Does it really so important? Looks like a mistake in `.git-blame-ignore-revs` can easily be fixed using another PR.

> Make sure to squash-merge this
We actually need to ignore only one of 3 commits in this PR, but with a separate PR we have no choice.